### DISCUSSION TITLE
Project Stopped working. "Rowy Run is not set up correctly" and cors errors

### DISCUSSION THREAD

**Discussion by Hedronmx** [AntlerVC/firetable#738] [Support / Q&A] [answered]
Hello i just cameback to rowy and I am finding out that the project is not working. I cannot access any of my tables.

![image](https://user-images.githubusercontent.com/26935602/182808052-73fd62af-d619-42bd-a682-7c35159f14db.png)

![image](https://user-images.githubusercontent.com/26935602/182808096-0b38eec0-ef8e-492b-abbf-c4f5a54db37a.png)

![image](https://user-images.githubusercontent.com/26935602/182808145-e4ca3989-d1d1-4742-b1b4-047bd7a5c5fb.png)

**Comment by harinij** [comment]
Hi @Hedronmx, Network error could potentially be internet issues. Also, can you share what version of Rowy Run does it say it is, when you click "Check for updates"?

**Reply by Hedronmx** [reply]
If i try to check for updates, i get a loading icon infinitely. I currently have no problems with my internet connection.

**Reply by harinij** [reply]
Could you go to `your_project_id.rowy.app/settings/debug` and click reset local Firestore cache. This should refresh the connection in case of any issues and let you verify. If that doesn't work then you can also manually paste the cloud run URL, and check for updates.  Let us know if that doesn't solve it either.
![Screen Shot 2022-08-05 at 12 13 11 pm](https://user-images.githubusercontent.com/307298/182987326-23a20a55-aa32-41ff-b902-1727216aed06.png)

**Reply by Hedronmx** [reply]
Resetted local firestore cache and the issue continues.

browsing to the cloud url gets me this
![image](https://user-images.githubusercontent.com/26935602/182988016-68bd687b-f6a8-4b02-be8d-ab1d680f4d00.png)

**Comment by Hedronmx** [comment, marked as answer]
It was some issue with google cloud! Its fixed now. Nothing wrong with rowy! Thank you so much

**Reply by gkodikara** [reply]
I have a similar issue - could you tell me how you fixed it? @Hedronmx
